### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by YoSTEALTH** [shuveb/loti#5]
`madvice(2)` link at https://unixism.net/loti/ref-liburing/setup_teardown.html?highlight=fork#c.io_uring_ring_dontfork is broken, it should be https://man7.org/linux/man-pages/man2/madvise.2.html

**Comment by YoSTEALTH** [comment]
Hi @shuveb hope you are doing well. Just wondering... have you given up on this project? It was really nice using https://unixism.net/loti/index.html docs

**Comment by shuveb** [comment]
Yeah @YoSTEALTH. It's on the shelf now since I'm fully tied up with running a startup!

**Comment by YoSTEALTH** [comment]
@shuveb  Aww... makes sense, well good luck
